recipes/giflib/5.2.x/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class GiflibConan(ConanFile):
+    name = "giflib"
+    description = "A library and utilities for reading and writing GIF images."
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    homepage = "http://giflib.sourceforge.net"
+    topics = ("conan", "giflib", "image", "multimedia", "format", "graphics")
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("%s-%s" % (self.name, self.version), self._source_subfolder)
+        
+        self._patch_sources()
```

Comment:
Move to build folder. Keep source folder untouched. You can change/remove the build folder easily, but build folder requires a new download.